mods/ts/weapons/ballisticweapons.yaml

```diff
@@ -108,6 +108,24 @@ RPGTower:
 		Explosions: large_explosion
 		ImpactSounds: expnew06.aud
 
+Jugg90mm:
```

Comment:
That's just the order in which I did things, I'll merge it into the main Juggernaut commit.